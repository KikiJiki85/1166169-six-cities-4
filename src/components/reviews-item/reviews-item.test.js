import React from "react";
import renderer from "react-test-renderer";
import ReviewsItem from "./reviews-item.jsx";

const review = {
  id: 1,
  text: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.`,
  rating: 4.3,
  date: `2019-04-24`,
  user: {
    avatar: `img/1.png`,
    id: 4,
    pro: false,
    name: `Max`,
  }
};

it(`Render ReviewsItem`, () => {
  const tree = renderer
    .create(
        <ReviewsItem
          key={review.id}
          review={review}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
