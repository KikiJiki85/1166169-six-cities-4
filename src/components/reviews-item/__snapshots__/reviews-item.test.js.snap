// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ReviewsItem 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="./img/avatar-angelina.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Angelina
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      2019-04-24
    </time>
  </div>
</li>
`;
